flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobExecutionITCase.java

```diff
@@ -37,6 +39,7 @@
 /**
  * Integration tests for job scheduling.
  */
+@Category(AlsoRunWithSchedulerNG.class)
 public class JobExecutionITCase extends TestLogger {
```

Comment:
This hotfix enabled NG scheduler testing for JobExecutionITCase which can pass at the moment.
It helps to verify the changes of FLINK-14040.

In this way we can have FLINK-14040 merged first. 
FLINK-14040 would be helpful when we are to annotate (and maybe need to fix) more tests. We can tell if the change works by checking both the PR test result(so that it works for legacy) and the cron job result(so that it works for NG).

Reply:
@GJL do you think it's Ok to merge this PR before annotating other MiniCluster tests?